site/triathlon/schwimmen/index.html

```diff
@@ -1,143 +1,137 @@
+          <h1>Schwimmen</h1>
+        </div>
+      </div>
+    </div>
+  </div>
 </div>
 <div class="row">
+  <div class="container">
     <div class="container">
-        <div class="container">
-            <!-- SIDEBAR vorlage -->
-            <h1>Sidebar</h1>
-            <div class="main-content">
-                <div class="sidebar">
-                    <div class="sticky">
-                        <div class="description">
-                            <h3>Auf dieser Seite</h3>
-                        </div>
-                        <ul>
-                            <li>
-                                <a href="../#jumper_triathlon_home">Home</a>
-                            </li>
-                            <li>
-                                <a href="../laufen/#jumper_triathlon_laufen"
-                                >Laufen</a
-                                >
-                            </li>
-                            <li class="active">
-                                <a href="../schwimmen/#jumper_triathlon_schwimmen"
-                                >Schwimmen</a
-                                >
-                            </li>
-                            <li>
-                                <a href="../radfahren/#jumper_triathlon_radfahren"
-                                >Radfahren</a
-                                >
-                            </li>
-                            <li>
-                                <a
-                                        href="../mobi-stabi-training/#jumper_triathlon_mobi-stabi-training"
-                                >Mobi-Stabi-Training</a
-                                >
-                            </li>
-                        </ul>
-                    </div>
-                </div>
+      <div class="activity-content">
+        <!-- Content/Text hier -->
+        <div class="content" id="jumper_triathlon_schwimmen">
+          <h2>Info</h2>
+          <p>
+            Schwimmen ist immer Montags im Lehrschwimmbecken Mergelstetten
+            (Silcherschule). Um 18:30 Uhr treffen sich die fortgeschrittenen
+            Schwimmer und um 19:30 Uhr jene, die an ihrer Technik feilen
+            möchten. Das Training dauert jeweils eine Stunde. Achtung, in den
+            Schulferien ist kein Montagsschwimmen. In den Sommermonaten
+            schwimmen wir zusätzlich im Waldbad Heidenheim auf der 50 Meter
+            Bahn. Treffpunkt ist mittwochs um 18.30 Uhr im Becken. Den Eintritt
+            übernimmt die Abteilung. Im Winter haben wir dienstags um 17:45 Uhr
+            eine 25 m-Bahn im Jurawell in Herbrechtingen. Hier müsst ihr den
+            Eintritt selbst übernehmen. Es empfiehlt sich eine Geldwertkarte.
+            Trainer sind Anka Röhr, Sven Müller und Michael Bacher. Je nach
+            Verfügbarkeit der Trainer findet auch mal nur ein freies Training
+            statt.
+            <br><br>
+          </p>
```

Comment:
No need to write custom css and HTML. 
I appreciate the effort but we wanted to have a unified design that looks the same through the whole website. The goal is to have a continious ui and user experience.
Also, the text break is weird on mobile
<img width="813" alt="Bildschirmfoto 2019-11-27 um 12 40 07" src="https://user-images.githubusercontent.com/11008743/69720735-93cda480-1113-11ea-914f-71adbb43f136.png">

Reply:
I agree with the weird text break - I'm trying to solve that

Reply:
fixed weird break